composer.json

```diff
@@ -5,44 +5,44 @@
     "license": "GPL-2.0-or-later",
     "require": {
         "acquia/lightning": "^4.0",
+        "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.0",
-        "drupal-composer/drupal-scaffold": "^2.0.0"
+        "drupal/core": "8.8.0-dev"
     },
     "require-dev": {
         "drush/drush": "^9.0"
     },
+    "conflict": {
+        "drupal/drupal": "*"
+    },
     "config": {
-        "process-timeout": 0
+        "process-timeout": 0,
+        "sort-packages": true
     },
     "extra": {
         "composer-exit-on-patch-failure": true,
         "enable-patching": true,
         "installer-paths": {
-            "docroot/core": [
-                "type:drupal-core"
-            ],
+            "docroot/core": ["type:drupal-core"],
             "docroot/libraries/{$name}": [
                 "type:drupal-library",
                 "type:bower-asset",
                 "type:npm-asset"
             ],
-            "docroot/modules/contrib/{$name}": [
-                "type:drupal-module"
-            ],
-            "docroot/profiles/contrib/{$name}": [
-                "type:drupal-profile"
-            ],
-            "docroot/themes/contrib/{$name}": [
-                "type:drupal-theme"
-            ],
-            "drush/contrib/{$name}": [
-                "type:drupal-drush"
-            ]
+            "docroot/modules/contrib/{$name}": ["type:drupal-module"],
+            "docroot/themes/contrib/{$name}": ["type:drupal-theme"],
+            "docroot/profiles/contrib/{$name}": ["type:drupal-profile"],
+            "drush/Commands/contrib/{$name}": ["type:drupal-drush"],
         },
         "installer-types": [
             "bower-asset",
             "npm-asset"
-        ]
+        ],
+        "composer-scaffold": {
+            "locations": {
```

Comment:
This can be removed, and instead require https://github.com/acquia/composer-scaffold

Reply:
D'oh... I think I might be wrong here. Maybe you used to be able to delegate the location definitions, but no longer. They have to be in the root composer.json.